snap-template/snapcraft.yaml

```diff
@@ -0,0 +1,30 @@
+name: my-snap  # the name of the snap
+version: 0  # the version of the snap
```

Comment:
I would suggest changing to 0.1 to point out that they can have decimals in the version.